packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -339,11 +339,11 @@ function CategoryWidgetUI(props) {
         direction='row'
         spacing={1}
         onClick={filterable ? onCategoryClick : () => {}}
-        onKeyDown={handleCategoryPress}
+        onKeyDown={filterable ? onCategoryPress : () => {}}
         selectable={filterable}
         unselected={unselected}
         name={data.name === REST_CATEGORY ? REST_CATEGORY : ''}
-        tabIndex={filterable && showAll ? 0 : -1}
```

Comment:
What's the impact of removing here 'showAll'? Are we sure about it?

Reply:
yes, otherwise the element is not reachable unless you have the search opened